### PR TITLE
node 16 production fork. Updated readme

### PR DESCRIPTION
**Pull request by tobiasnteireho**
node 12 goes EOL end of April. Upgrades to 16 and upgrades steal to 2

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -187,5 +189,8 @@
         }
       ]
     ]
+  },
+  "volta": {
+    "node": "16.13.1"
```
Comment (mitch-mitchel) [inline package.json]: I think the other repos have node pinned at 16.13.2 right now, if you want bump this

**Comment by tobiasnteireho** [comment]
fix #115 and fix #89

File: package.json (hunk 1 above)
Comment (tobiasnteireho) [reply package.json]: good catch. it looks like author and viewer is pinned to 16.14.2. Changed in develop branch too as that is where this file is from

**Comment by mitch-mitchel** [comment]
👍

**Pull request merged**